tests/tracking/test_fluent.py

```diff
@@ -153,7 +153,7 @@ def is_from_run(active_run, run):
 
 def test_start_run_defaults(empty_active_run_stack):
 
-    mock_experiment_id = mock.Mock()
+    mock_experiment_id = str(mock.Mock())
```

Comment:
er, str of a mock? That doesn't seem right.

Reply:
It was not. I was able to get around it, the test did not like the internal type change, I only make the change for int now, which is how I went around this.